test/tsconfig.json

```diff
@@ -2,6 +2,12 @@
   "extends": "@salesforce/dev-config/tsconfig-test",
   "include": ["./**/*.ts"],
   "compilerOptions": {
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "baseUrl": "..",
```

Comment:
will compile using the top-level imports of this repo (when they satisfy dependencies) instead of transient dependencies' versions.

This matches the top-level `tsconfig` so we don't have "project builds but tests don't compile"